### DISCUSSION TITLE
Popup with permission issue

### DISCUSSION THREAD

**Discussion by mittygritty** [JonathanTreffler/backblaze-personal-wine-container#50] [Q&A]
Hi, great container! I've noticed this pop up keeps coming, the upload still seems to be working, but Im not sure if it causes issues etc:

https://ibb.co/0JwvRqp

**Comment by omgitstyler** [comment]
I get the same error message. Is there a permission setting we can change to prevent the error?

**Comment by devanteweary** [comment]
Same issue here guys.

**Comment by Nexus-Unknown** [comment]
Same for me

**Comment by csharpengineer** [comment]
I read in an issue post that you can just ignore the message.  I personally click on the backblaze client window so the message goes to the background and leave it there.
